packages/@aws-cdk/toolkit-lib/lib/actions/refactor/index.ts

```diff
@@ -14,4 +14,13 @@ export interface RefactorOptions {
    * @default - all stacks
    */
   stacks?: StackSelector;
+
+  /**
+   * The absolute path to a file that contains a list of
+   * resources to skip during the refactor. The file should
+   * be in JSON format and contain an array of _destination_
+   * logical IDs, that is, the logical IDs of the resources
+   * as they would be after the refactor.
+   */
+  skipFile?: string;
```

Comment:
I'm looking for a better name. If anyone has any suggestions, let me know.

Reply:
For toolkit-lib, this should be an array. We can make the read from file a CLI feature.

Reply:
> I'm looking for a better name. If anyone has any suggestions, let me know.

`exclude`?